website/content/docs/secrets/kv/kv-v2/setup.mdx

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs
+page_title: Set up the key/value v2 plugin
+description: >-
+   Enable and configure the key/value v2 plugins to store arbitrary static
+   secrets in Vault.
+---
+
+# Set up the key/value v2 plugin
+
+Use `vault secrets enable` to enable an instance of the `kv` plugin. To specify
+version 2, use the `-version` flag or specific `kv-v2` as the plugin type:
+
+Additionally, when running a dev-mode server, the v2 `kv` secrets engine is enabled by default at the
+path `demo/` (for non-dev servers, it is currently v1). It can be disabled, moved, or enabled multiple times at
+different paths. Each instance of the KV secrets engine is isolated and unique.
+
+
+## Before you start 
+
+- **You must have permission to update ACL policies**.
+- **You must have permission to enable plugins**.
+
+
+
+## Step 1: Enable the plugin
+
+
+<Tabs>
+
+<Tab heading="CLI" group="cli">
+
+Use `vault secrets enable` to establish a new instance of the `kv` plugin.
+
+To specify key/value version 2, use the `-version` flag or use `kv-v2` as the
+plugin type.
+
+
+**Option 1**: Use the `-version` flag:
+
+```shell-session
+$ vault secrets enable -path <mount_path> -version=2 kv
+```
+
+**Option 2**: Use the `kv-v2` plugin type:
+
+```shell-session
+$ vault secrets enable -path <mount_path> kv-v2
+```
+
+</Tab>
+
+<Tab heading="GUI" group="gui">
+
+@include 'gui-page-instructions/enable-secrets-plugin.mdx'
+
+- Select the "KV" plugin.
+
+- Enter a unique path for the plugin and provide the relevant configuration
+  data.
+
+</Tab>
+
+</Tabs>
+
+
+
+## Step 2: Create an ACL policy file
```

Comment:
Maybe you can mention the [`-output-policy` flag usage](https://developer.hashicorp.com/vault/docs/commands#construct-the-required-vault-policy) somewhere? I helps users to write/verify policies.

Reply:
That's a great idea. I'll add that today